### DISCUSSION TITLE
Edits lost if encoding in progress

### DISCUSSION THREAD

**Discussion by hquimby** [mediacms-io/mediacms#1101] [Q&A]
When I upload a new video and after processing has started, I use the edit media button to change the Title, assign the Category and set it as Featured.  The changes appear to save as expected.  But when media encoding is complete, the Title reverts to the original, the Featured checkmark is unchecked, and only the Category change persists.

Is this a defect?  Should the edit media button be enabled while encoding?  Or do we need to wait for all encoding to complete before making edits?

**Comment by mgogoulos** [comment]
https://github.com/mediacms-io/mediacms/issues/1104
